src/stories/components/Review/styles.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Review
+
+.review {
+  padding: $spacing-1 $spacing-4;
+  max-width: rem(600px);
+  font-size: $font-size-sm
+}
+
+.review__recipe {
+  font-size: $font-size-sm;
+  font-family: $header-font-family;
+  font-weight: 700;
+  letter-spacing: rem(1px);
+
+  text-transform: uppercase;
+  // TODO: Change to sand-dark, I saw there was already on created
+  //   so will wait until that PR is merged then use it.
+  color: $sand;
+}
+
+.review__title {
+  font-family: $header-font-family;
+  font-size: $font-size-lg;
+  font-weight: 700;
+}
+
+.review__sub-header {
+  width: 80%;
+  font-size: $font-size-xs;
+  line-height: $line-height-loose;
+}
+
+.review__sub-header-item {
+  border-color: black;
+  border-width: 1px;
+  border-style: none solid none none;
+  padding: rem(2px) $spacing-2;
+  &:first-child {
+    padding-left: rem(4px);
+  }
+  &:last-child {
+    border-style: none;
+  }
+}
+
+
+.review__content {
+  margin-top: $spacing-1;
+}
+
+.review__view-recipe {
+  text-align: right;
+  vertical-align: middle;
+  
+  a {
+    text-decoration: none;
+  }
+}
+
+.pad-left {
+  margin-left: $spacing-3;
+}
```

Comment:
What is the need of this?

Reply:
Oop good catch, absolutely nothing is the need, haha.